src/test/java/com/serenitydojo/WhenWorkingWithMaps.java

```diff
@@ -71,7 +80,7 @@ public void containsKey() {
                 "hamster", LETTUCE);
 
         // TODO: Check that the map contains a key of "dog"
-        boolean containsDog = false;
+        boolean containsDog = true;
```

Comment:
i think you are supposed to use the containsKey method here https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/util/Map.html#containsKey(java.lang.Object)

Reply:
Okay, rechecking my code

Reply:
```suggestion
        boolean containsDog = favoriteFood.containsKey("dog");
```